### PR TITLE
changes postgres instructions to fix issue with default user role

### PR DESCRIPTION
**Pull request by katcosgrove**
Current postgres setup instructions provide instructions for forcing user login as the default postgres user, but leave the machine without a role or database for the default user. This causes a `fatal: role <user> does not exist` error to fire whenever commands that rely on the default user's access to postgres are executed, like pushing a database to heroku. This adjusts the instructions to include creating a role for the default user, granting it superuser privileges, and creating a default database.

### CODE REVIEW

File: setup_local_env.md [hunk 1]
```diff
@@ -34,14 +34,19 @@ For both Windows and Linux users, please follow the default installation instruc
...
+- In your SQL shell, type the following: `CREATE ROLE your-username-here WITH LOGIN PASSWORD 'your-password-here';`, replacing "your-password-here" with whatever you want it to be. Remember that your password must be wrapped in quotes. The username should not be wrapped in quptes. *Don't forget the semicolon*.
+- If successful, you will receive the feedback `CREATE ROLE`.
+- Now we need to grant that user administrative control. In your SQL shell, type the following: `ALTER ROLE your-username-here WITH superuser;`, replacing "your-username-here" with the username you created a role for in the previous step.
+- If successful, you will receive the feedback `ALTER ROLE`.
+- In another terminal window, outside of the SQL shell, type the following: `createdb your-username-here`, replacing "your-username-here" with the username used in the SQL shell. You may not receive feedback from this command.
```
Comment (mnfmnfm) [inline setup_local_env.md]: `createdb` is not a consistently-available util; it's safer to do this within the SQL shell as `CREATE DATABASE your-username-here;`

Comment (katcosgrove) [reply setup_local_env.md]: Requested changes made in next commit :)

File: setup_local_env.md [hunk 2]
```diff
@@ -34,14 +34,20 @@ For both Windows and Linux users, please follow the default installation instruc
...
-- If successful, you will receive the feedback `ALTER ROLE`. Hooray! Continue to [final steps!](#final-steps).
+- If you were not prompted for a default user or password, we will set one using psql. If you type `\du`, you can get a list of users associated with PostgreSQL. You should see a single user, `postgres`. You will need to set up a new role for your machine's default user. This is the username that appears at the beginning of your terminal prompt, and when you log into your machine.
+- In your SQL shell, type the following: `CREATE ROLE your-username-here WITH LOGIN PASSWORD 'your-password-here';`, replacing "your-password-here" with whatever you want it to be. Remember that your password must be wrapped in quotes. The username should not be wrapped in quptes. *Don't forget the semicolon*.
```
Comment (brookr) [inline setup_local_env.md]: Typo: `quptes`

**Comment by bnates** [comment]
@katcosgrove mind taking a look/fixing the typo item outlined above?

**Comment by katcosgrove** [comment]
Typo fixed, requested changes made

**Comment by brookr** [comment]
@katcosgrove We aren't seeing the commits on the branch yet... 
`Add more commits by pushing to the sql-setup-fix branch on codefellows/code-301-prework.`

**Comment by katcosgrove** [comment]
@brookr Updated final steps and caught a grammatical error in wsl

**Pull request merged**